### ISSUE TITLE
expose codes from THREAD_TABLE

### ISSUE THREAD

**Issue by hrgdavor** [receter/jscad-threadlib#1]
It would be nice to be able to get all available codes,

that way a jscad script can generate paramater select box to play with all available threads

**Comment by receter** [comment]
I could easily export the THREAD_TABLE this would allow to do:

```javascript
import {
  THREAD_TABLE,
} from "jscad-threadlib";
const allThreadDesignators = THREAD_TABLE.map(([threadDesignator]) => threadDesignator);
```

What do you think?

**Comment by hrgdavor** [comment]
yup, that would be even better :)

**Comment by receter** [comment]
I merged the change and released a new version v0.0.3

**Comment by hrgdavor** [comment]
cool, it is already visible on jscad.app . This opens nice posibilities to present the different threads.
